ddtrace/appsec/_iast/reporter.py

```diff
@@ -75,9 +76,18 @@ class Vulnerability(NotNoneDictable):
     evidence: Evidence
     location: Location
     hash: int = dataclasses.field(init=False, compare=False, hash=("PYTEST_CURRENT_TEST" in os.environ), repr=False)
+    stackId: Optional[str] = None
 
     def __post_init__(self):
+        # avoid circular import
+        from ddtrace.appsec._iast._iast_request_context import get_iast_stacktrace_id
+
         self.hash = zlib.crc32(repr(self).encode())
+        stacktrace_id = get_iast_stacktrace_id()
+        if stacktrace_id:
+            str_id = str(stacktrace_id)
+            if report_stack(stack_id=str_id, ep_or_iast=False):
```

Comment:
What I mean above, if we had a constant `IAST_NAMESPACE` with value `"vulnerability"`here we would have something like:
```suggestion
            if report_stack(stack_id=str_id, namespace=IAST_NAMESPACE):
```

Reply:
yes, I like the suggestion. I'll do it.

Reply:
done.